py3-ruamel.yaml.clib.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: py3-ruamel.yaml.clib
```

Comment:
I think `.`s in the package name might confuse some things that parse package+versions from apk filenames. Can we make this `py3-ruamel-yaml-clib` and rename this file accordingly? I think that will also address the wolfictl issue you found.

If this works we should enforce this naming rule in a linter in this repo to prevent future instances.

Thanks for this contribution! 🎉